dds/idl/marshal_generator.cpp

```diff
@@ -2697,7 +3212,7 @@ bool marshal_generator::gen_struct(AST_Structure* node,
         // TODO (sonndinh): Integrate with try-construct for when the stream
         // ends before some fields on the reader side get their values.
         if (exten == extensibilitykind_appendable) {
-          expr += "  if (strm.encoding().xcdr_version() == Encoding::XCDR_VERSION_2 &&\n";
+          expr += "  if (strm.encoding().xcdr_version() == Encoding::XCDR_VERSION_2 &&";
           expr += "      strm.pos() - start_pos >= total_size) {\n";
```

Comment:
This would result in

```C++
  if (strm.encoding().xcdr_version() == Encoding::XCDR_VERSION_2 &&      strm.pos() - start_pos >= total_size) {
```

Put the newline back or fix the spacing.